### ISSUE TITLE
[Bug]: Podcast episodes still downloading after scheduled downloads disabled

### ISSUE THREAD

**Issue by undaunt** [advplyr/audiobookshelf#2277]
### Describe the issue

I disabled the scheduled download of a subset of my podcasts, but they seem to continue to check for updated episodes and download them. Here are some logs and my settings for 538 politics:

```
11/02/2023
09:00:50 AM
[2023-11-02 16:00:50] INFO: [PodcastManager] runEpisodeCheck: "FiveThirtyEight Politics" | Last check: Thu Nov 02 2023 15:00:49 GMT+0000 (Coordinated Universal Time) | No latest episode
11/02/2023
10:00:49 AM
[2023-11-02 17:00:49] INFO: [PodcastManager] runEpisodeCheck: "FiveThirtyEight Politics" | Last check: Thu Nov 02 2023 16:00:50 GMT+0000 (Coordinated Universal Time) | No latest episode
11/02/2023
10:00:49 AM
[2023-11-02 17:00:49] INFO: [PodcastManager] Found 1 new episodes for podcast "FiveThirtyEight Politics" - starting download
11/02/2023
11:00:51 AM
[2023-11-02 18:00:51] INFO: [PodcastManager] runEpisodeCheck: "FiveThirtyEight Politics" | Last check: Thu Nov 02 2023 17:00:49 GMT+0000 (Coordinated Universal Time) | Latest episode pubDate: Thu Nov 02 2023 16:31:53 GMT+0000 (Coordinated Universal Time)
11/02/2023
12:00:48 PM
[2023-11-02 19:00:48] INFO: [PodcastManager] runEpisodeCheck: "FiveThirtyEight Politics" | Last check: Thu Nov 02 2023 18:00:51 GMT+0000 (Coordinated Universal Time) | Latest episode pubDate: Thu Nov 02 2023 16:31:53 GMT+0000 (Coordinated Universal Time)
11/02/2023
01:00:48 PM
[2023-11-02 20:00:48] INFO: [PodcastManager] runEpisodeCheck: "FiveThirtyEight Politics" | Last check: Thu Nov 02 2023 19:00:49 GMT+0000 (Coordinated Universal Time) | Latest episode pubDate: Thu Nov 02 2023 16:31:53 GMT+0000 (Coordinated Universal Time)
```

![image](https://github.com/advplyr/audiobookshelf/assets/31376520/41910ff0-b08f-4dca-bbcd-39abdbfea199)

### Steps to reproduce the issue

1. Disable scheduled downloads of a podcast.
2. Wait for an episode to be released and/or review logs to verify podcasts are still being checked for episodes.

### Audiobookshelf version

2.5.0

### How are you running audiobookshelf?

Docker

**Comment by advplyr** [comment]
Thanks, I just fixed this bug. It may also explain some user reports that the podcast schedule didn't run.

**Comment by advplyr** [comment]
Fixed in [v2.6.0](https://github.com/advplyr/audiobookshelf/releases/tag/v2.6.0)
